subprojects/docs/src/docs/userguide/upgrading_version_4.adoc

```diff
@@ -634,13 +711,34 @@ As part of the stabilization effort in Gradle 4.8, we are deprecating this behav
 
 The new, stable behavior can be switched on by adding the following to your settings file:
 
-    enableFeaturePreview('STABLE_PUBLISHING')
+====
+[.multi-language-sample]
+=====
+.settings.gradle
+[source, groovy]
+----
+enableFeaturePreview('STABLE_PUBLISHING')
```

Comment:
IMO we should always suggest `"` in Groovy, but we already had that discussion and I will just let this go.